backend/corpora/common/utils/tissue_ontology_mapping.py

```diff
@@ -1,5 +1,5 @@
 # Generated by
-# https://github.com/chanzuckerberg/single-cell-curation/blob/main/notebooks/compute_tissue_ancestor_mapping.ipynb
+# https://github.com/chanzuckerberg/single-cell-curation/blob/main/notebooks/compute_tissue_and_cell_type_mappings.ipynb
```

Comment:
I have consolidated tissue and cell type notebooks; corresponding PR will be ready shortly.